doc/index.rst

```diff
@@ -1,39 +1,47 @@
-PennyLane Qiskit Plugin
+PennyLane-Qiskit Plugin
 #######################
 
 :Release: |release|
 :Date: |today|
```

Comment:
Would this be something that we'd like to keep?

Reply:
You are right, data is superfluous. I deleted that line...Release number is still essential IMO.